### PR TITLE
[Fix] Accuracy changed from array to float

### PR DESCRIPTION
**Pull request by kotekjedi**

### CODE REVIEW

File: shifthappens/tasks/imagenet_r/imagenet_r.py [hunk 1]
```diff
@@ -78,8 +78,18 @@ def _evaluate(self, model: sh_models.Model) -> TaskResult:
...
             accuracy=accuracy, summary_metrics={Metric.Robustness: "accuracy"}
         )
+
+
+if __name__ == "__main__":
+    from shifthappens.models.torchvision import ResNet18
+
+    sh_benchmark.evaluate_model(
+        ResNet18(device="cuda", max_batch_size=128), "../../../../test_data"
+    )
```
Comment (kotekjedi) [inline shifthappens/tasks/imagenet_r/imagenet_r.py:95]:
```suggestion
        )
```

File: shifthappens/tasks/imagenet_c/imagenet_c.py [hunk 2]
```diff
@@ -449,7 +449,7 @@ def _evaluate(self, model: sh_models.Model) -> TaskResult:
 
         return TaskResult(
             **results,
-            accuracy=np.mean(accuracies).item(),
-            mce=np.mean(mces).item(),
+            accuracy=np.mean(np.array(accuracies)),
```
Comment (zimmerrol) [inline shifthappens/tasks/imagenet_c/imagenet_c.py:452]: That doesn't look right, as we already take the mean in L102.

**Review by zimmerrol — CHANGES_REQUESTED** [review]
Thanks for this PR. Can you please explain why you propose these changes/what they are meant to fix?

File: shifthappens/tasks/imagenet_c/imagenet_c.py (hunk 2 above)
Comment (kotekjedi) [reply shifthappens/tasks/imagenet_c/imagenet_c.py:452]: Here `accuracies` is an array of the accuracy values for every task. So we take the mean of these accuracies in L452.

**Comment by kotekjedi** [comment]
@zimmerrol At the moment, in L105, `accuracy` is an array of True and False values. I think it is a bug because we require the `metrics` field in `TaskResult` to be a `**metrics: Union[float, int]`.

**Review by zimmerrol — APPROVED** [review]

**Pull request merged**